ciao-controller/quotas.go

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) 2017 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"github.com/01org/ciao/ciao-controller/internal/datastore"
+	"github.com/01org/ciao/ciao-controller/internal/quotas"
+	"github.com/01org/ciao/ciao-controller/types"
+	"github.com/01org/ciao/payloads"
+	"github.com/pkg/errors"
+)
+
+func (c *controller) UpdateQuotas(tenantID string, qds []types.QuotaDetails) {
+	c.qs.Update(tenantID, qds)
+	for i := range qds {
+		c.ds.UpdateQuota(tenantID, qds[i])
```

Comment:
Can't this call to UpdateQuota fail? If so, you are not checking the return and could wind up with persistent data out of sync with quota service.